clang/lib/CodeGen/CodeGenFunction.h

```diff
@@ -5149,6 +5152,12 @@ class CodeGenFunction : public CodeGenTypeCache {
                            SourceLocation ArgLoc, AbstractCallee AC,
                            unsigned ParmNum);
 
+  /// EmitWriteback - Emit callbacks for function.
+  void EmitWritebacks(CodeGenFunction &CGF, const CallArgList &Args);
+
+  void EmitWriteback(CodeGenFunction &CGF,
+                     const CallArgList::Writeback &writeback);
```

Comment:
Was it necessary to expose `EmitWriteback`?  It seems you would only need `EmitWritebacks`.